server/src/main/java/org/elasticsearch/common/logging/NodeAndClusterIdConverter.java

```diff
@@ -19,76 +19,60 @@
 
 package org.elasticsearch.common.logging;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.core.LogEvent;
 import org.apache.logging.log4j.core.config.plugins.Plugin;
 import org.apache.logging.log4j.core.pattern.ConverterKeys;
 import org.apache.logging.log4j.core.pattern.LogEventPatternConverter;
 import org.apache.logging.log4j.core.pattern.PatternConverter;
-import org.apache.lucene.util.CloseableThreadLocal;
-import org.elasticsearch.cluster.ClusterChangedEvent;
-import org.elasticsearch.cluster.ClusterStateListener;
-import org.elasticsearch.cluster.node.DiscoveryNode;
-import org.elasticsearch.common.util.LazyInitializable;
 
 import java.util.Locale;
 import java.util.concurrent.atomic.AtomicReference;
 
-
+/**
+ * Pattern converter to format the node_and_cluster_id variable into a json fields node.id and cluster.uuid
```

Comment:
Nits:

* `node.id`-> `<code>node.id</code>` (similarly for `cluster.uuid`)
* Missing punctuation mark